me.js

```diff
@@ -0,0 +1,70 @@
+
+const page = document.querySelector('body');
+page.style.fontFamily = "Arial, sans-serif" ;
+
+const firstName = document.getElementById('nickname');
+firstName.innerHTML = "Lama";
+
+const favourite = document.getElementById('favorite');
+favourite.innerHTML = "Parrot";
+
+const town = document.getElementById('hometown');
+town.innerHTML = "Riyadh";
+
+
+document.getElementsByTagName('li').className = "listitem";
```

Comment:
This line will return an array of all li elements so you can loop through them and add the class name
```suggestion
let listItems = document.getElementsByTagName('li');
    for (let i = 0; i < listItems.length; i++) {
        const element = listItems[i];
        element.className = 'listitem'
        
    }
```